documentation/resource_apache2_config.md

```diff
@@ -0,0 +1 @@
+# config
```

Comment:
Should we have a bit more documentation here?

Reply:
Yep but that's for #572